.github/workflows/postsubmit.yml

```diff
@@ -33,3 +33,25 @@ jobs:
               python3 test/renderdiff/src/update_golden.py --branch=${GOLDEN_BRANCH} \
                   --merge-to-main --filament-tag=${COMMIT_HASH} --golden-repo-token=${GH_TOKEN}
           fi
+
+  update-docs:
+    name: update-docs
+    runs-on: 'ubuntu-24.04-4core'
+    steps:
+      - uses: actions/checkout@v4.1.6
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/linux-prereq
+      - id: get_commit_msg
+        uses: ./.github/actions/get-commit-msg
+      - name: Prerequisites
+        run: pip install selenium
+      - name: Run update script
+        env:
+          GH_TOKEN: ${{ secrets.FILAMENTBOT_TOKEN }}
```

Comment:
I think we should avoid using the Filament bot token. Github now provides you with a token to use in workflows:

https://docs.github.com/en/actions/how-tos/security-for-github-actions/security-guides/automatic-token-authentication